src/Products/Signature/Controllers/SignatureApiController.cs

```diff
@@ -953,7 +979,7 @@ private void SignDigital(string documentType, string password, SignatureDataEnti
         /// <param name="documentType">string</param>
         /// <param name="signaturesData">SignatureDataEntity</param>
         /// <param name="signsCollection">SignatureOptionsCollection</param>
-        private void SignImage(string documentType, SignatureDataEntity signaturesData, SignatureOptionsCollection signsCollection)
+        private void SignImage(string documentType, SignatureDataEntity signaturesData, List<SignOptions> signsCollection)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'SignImage' a static method.](https://app.codacy.com/gh/groupdocs-signature/GroupDocs.Signature-for-.NET-MVC/pullRequest?prid=5325146)